OperatorOverloading/OperatorOverloading/Money.cs

```diff
@@ -0,0 +1,80 @@
+﻿namespace OperatorOverloading
+{
+    public class Money
+    {
+        public int Amount { get; set; }
+        public CurrencyType CurrencyType { get; }
+
+        public Money(int amount, CurrencyType currencyType)
+        {
+            this.Amount = amount;
+            this.CurrencyType = currencyType;
+        }
+
+        public static implicit operator double(Money moneyInstance)
+        {
+            return (double)moneyInstance.Amount;
+        }
+        public static explicit operator string(Money moneyInstance)
+        {
+            return moneyInstance.Amount.ToString();
+        }
+        public static implicit operator Money(double amount)
+        {
+            return new Money((int)amount, CurrencyType.UAN);
+        }
+        public static explicit operator Money(string amount)
+        {
+            return new Money(int.Parse(amount), CurrencyType.UAN);
+        }
+
+        public static Money operator ++(Money moneyIstance)
```

Comment:
i believe the best name will be just `instance` or `obj` and anyway you have type in `moneyIstance`, correct one `moneyInstance`

Reply:
Mistyped((( And, yeah, instance would be better name for this variable